index.js

```diff
@@ -31,9 +31,10 @@ function create (config = {}) {
       // Initialize all scripts and stash command scripts
       const commands = new Map()
       scripts.forEach(script => {
-        const handler = script(bot, config)
-        if (script.command && handler) {
-          commands.set(script.command, handler)
+        const plugin = values(values(script)[0])[0]
```

Comment:
@jamen could you check this maybe? it's pretty bad and I think it breaks your initial intention.

I just want a solution to load all index.js files inside `./plugins/` automatically.